azure/group.sh

```diff
@@ -24,11 +24,11 @@ fn azure_group_get_names() {
 # azure_group_exists returns "0" if a resource group
 # already exists, "1" otherwise.
 fn azure_group_exists(name) {
-        groups <= azure_group_get_names()
-        for group in $groups {
-            if $group == $name {
-                return "0"
-            }
-        }
-        return "1"
+	group_name <= az group show --name $name | jq -r ".name"
```

Comment:
Nash aborts if some program fails (return $status != 0) and user didn't check the status.
I don't know if this is the case, but if az program fails if group doesn't exists then nash will abort at this statement also.
You can just rely on $status variable to know if the group exists or ask Nash to ignore the status to not abort.

```sh
fn azure_group_exists(name) {
    # only checking status
    _, status <= az group show --name $name 
    return $status
}
```
or like you did but asking Nash to ignore error:
```sh
fn azure_group_exists(name) {
    groupName, _ <= az group show --name $name | jq -r ".name"
    if $groupName == $name {
        return "0"
    }
    return "1"
}
```